docs/modules/ROOT/pages/tmail-backend/jmap-extensions/calendarEventParsing.adoc

```diff
@@ -0,0 +1,137 @@
+= Calendar event parse
+:navtitle: calendar event parse
+
+Allows users to parse calendar events for getting event information from the blob id of the calendar file (.ics).
+
+== Additions to the capability object
+
+Servers supporting the calendar event parse extension need
+to advertise it through the session capabilities:
+....
+com:linagora:params:calendar:event
+....
+
+The associated object is empty.
+
+== CalendarEvent/parse
+
+This method bases on:
+- Parsing a Received MDN method as described in the RFC-9007
+- JMAP CalendarEvent RFC-8620
+- iCalendar message RFC-5545
+
+The request object requires the following properties:
+
+- `accountId`: accountId of action user
+- `blobIds`: an array blobId of calendar which user wants to parse.
```

Comment:
Can we present properties more in a jmap spec way?

```
- *accountId*: `Id` The id of the account to use.
- *blobIds*: `[Id]` An array of calendar blob ids that the user wants to parse.
```

Same for the response properties